pkg/comp-functions/functions/vshnnextcloud/files/000-default.conf

```diff
@@ -0,0 +1,5 @@
+<VirtualHost *:8080>
```

Comment:
Do we need this file? can't we use the default one from the image

Reply:
The default one binds to Port 80. That is the only reason why I have this file, as we can't bind to Port 80 on Openshift without additional SCCs.